### PR TITLE
Add user interactions

### PR DESCRIPTION
**Pull request by joanb**

### CODE REVIEW

File: app/src/main/java/com/theopensourcefamily/chessclock/ClocksActivity.kt [hunk 1]
```diff
@@ -2,11 +2,19 @@ package com.theopensourcefamily.chessclock
...
 import androidx.appcompat.app.AppCompatActivity
+import com.jakewharton.rxbinding3.view.clicks
+import io.reactivex.Observable
 import kotlinx.android.synthetic.main.activity_clocks.*
 
-class ClocksActivity : AppCompatActivity(), GameView {
+class ClocksActivity : AppCompatActivity(), ClocksView {
 
-  val presenter = GamePresenter() // maybe DI here, with koin or wathever
+  val presenter = ClocksPresenter() // maybe DI here, with koin or wathever
```
Comment (SergiLlamasGuerrero) [inline app/src/main/java/com/theopensourcefamily/chessclock/ClocksActivity.kt]: Great idea! What do you think about adding this here or in another PR, I don't care.

Comment (SergiLlamasGuerrero) [inline app/src/main/java/com/theopensourcefamily/chessclock/ClocksActivity.kt]: Great idea! What do you think about adding koin here or in another PR, I don't care.

File: app/src/main/java/com/theopensourcefamily/chessclock/ClocksPresenter.kt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package com.theopensourcefamily.chessclock
+
+import io.reactivex.disposables.CompositeDisposable
+
+class ClocksPresenter {
```
Comment (SergiLlamasGuerrero) [inline app/src/main/java/com/theopensourcefamily/chessclock/ClocksPresenter.kt]: We will use MVP pattern? Maybe it's MVVM pattern, but I never worked with MVVM before. Anyway we can talk about what pattern should we use.

File: app/src/main/java/com/theopensourcefamily/chessclock/ClocksActivity.kt (hunk 1 above)
Comment (joanb) [reply app/src/main/java/com/theopensourcefamily/chessclock/ClocksActivity.kt]: Yes I implemented this with Koin, but then i was thinking that maybe we don't ever need an instance of the presenter in the activity... and I'm not sure how to do this "scoped" injection with Koin and I didn't want to get stuck with this in this PR 😅  
We can create an issue just to do this! and this issue wont block any further implementations 😋

File: app/src/main/java/com/theopensourcefamily/chessclock/ClocksPresenter.kt (hunk 2 above)
Comment (joanb) [reply app/src/main/java/com/theopensourcefamily/chessclock/ClocksPresenter.kt]: I was thinking on going fully reactive with MVI pattern. This is kind of an experiment to me also, so let's see what happens! :D!

File: app/src/main/java/com/theopensourcefamily/chessclock/ClocksActivity.kt (hunk 1 above)
Comment (SergiLlamasGuerrero) [reply app/src/main/java/com/theopensourcefamily/chessclock/ClocksActivity.kt]: Ok! I agree! :)

File: app/src/main/java/com/theopensourcefamily/chessclock/ClocksPresenter.kt (hunk 2 above)
Comment (SergiLlamasGuerrero) [reply app/src/main/java/com/theopensourcefamily/chessclock/ClocksPresenter.kt]: Let's play! 😋

**Pull request merged**